conf/configuration.yaml

```diff
@@ -118,3 +118,9 @@ arlas-ogc:
 arlas-csw:
   openSearchDescription: ${ARLAS_CSW_OPENSEARCH_DESCRIPTION:-Geo-BigData Collection Catalog}
   openSearchShortName: ${ARLAS_CSW_OPENSEARCH_SHORTNAME:-Geo-Catalog}
+
+########################################################
+############ OPENSEARCH                  ###############
+########################################################
+opensearch:
+  url-template-prefix: ${ARLAS_OPENSEARCH_URL_TEMPLATE:-http://ARLAS-HOST/arlas/explore/[collection]/_search} # Only ARLAS_HOST and the endpoint '_search' can be modified. '/arlas/explore/[collection]/' should remain as it is.
```

Comment:
I guess there is a missunderstanding about this point.
Devops that uses ARLAS should be able to use rewrite rules to expose their own url schemes. As a result, the whole `url-template-prefix` can be modified.